app/components/key_pair_generator_component.rb

```diff
@@ -0,0 +1,11 @@
+class KeyPairGeneratorComponent < BaseComponent
+  attr_reader :location
+
+  def initialize(location:)
+    @location = location
+  end
+
+  def render?
+    AbTests::KEY_PAIR_GENERATION.bucket(SecureRandom.uuid) == :key_pair_group
+  end
```

Comment:
While I don't think it's wrong to consider this as an A/B test, I wonder if it's worth all the overhead (both runtime performance and implementation maintenance) when we don't have a deterministic discriminator, vs. just using basic randomization.

```suggestion
  def render?
    rand(1...100) <= IdentityConfig.store.key_pair_generation_percent
  end
```

Reply:
Sure, but again this is temporary so at this point I'm going to leave it. Running the existing randomization 100 times took less than 1 ms on my laptop. ¯\_(ツ)_/¯